doc/02-Twig-Integration.markdown

```diff
@@ -284,6 +286,7 @@ Twig integration reference
 * `knp_menu_get('menuName' [, ['Path', 'To', 'Item'], ['options']])`: retrieve an item of the menu
 * `knp_menu_render('menuName' [, ['options'], 'rendererName'])`: render a menu
 * `knp_menu_get_breadcrumbs_array('menuName' [, 'subItem'])`: get an array that represent the breadcrumbs of the current page (according to the menu)
+* `knp_menu_get_current_item('menuName')`: retrieve the current item (according to the menu)
```

Comment:
it would be cool to put your example from the PR description here after the list. it becomes more understandable with a concrete example imho